app/src/main/java/de/mvhs/android/zeiterfassung/ExportProgressDialog.java

```diff
@@ -0,0 +1,42 @@
+package de.mvhs.android.zeiterfassung;
+
+import android.app.Activity;
+import android.support.v7.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+
+public class ExportProgressDialog extends DialogFragment {
```

Comment:
Die Implementierung ist in diesem Fall zwar nicht notwendig, da ein ProgressDialog bereits existiert, aber sehr schöne Übung, um auch Dialoge mit eigenen Layout und Logik zu erstellen. Sehr schön!!!